x/stakeibc/client/cli/query_delegation.go

```diff
@@ -1,36 +0,0 @@
-package cli
-
-import (
-	"context"
-
-	"github.com/Stride-Labs/stride/x/stakeibc/types"
-	"github.com/cosmos/cosmos-sdk/client"
-	"github.com/cosmos/cosmos-sdk/client/flags"
-	"github.com/spf13/cobra"
-)
-
-func CmdShowDelegation() *cobra.Command {
```

Comment:
I think there were certain CLI tests that were scaffolded because ignite assumed we were going to initialize them at genesis (e.g. Delegation and MinValidatorRequirements data types). Since we don't actually need them at genesis, I think we can remove these tests